spec/requests/past_court_dates_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "/casa_cases/:casa_case_id/past_court_dates/:id", :disable_bullet, type: :request do
```

Comment:
We need to start un-breaking bullet... we have way too many n+1s 
https://github.com/flyerhzm/bullet